test/e2e/media.spec.ts

```diff
@@ -167,7 +176,8 @@ describe('Media Metadata endpoints', () => {
     it('should return an episode by imdbid', async() => {
       const body = JSON.stringify({ imdbid: 'tt3388032' });
       const response: any = await got.post(`${appUrl}/api/media/imdbid`, { responseType: 'json', headers: { 'content-type': 'application/json' }, body });
-      expect(response.body.episodeTitle).toEqual('Proof of Concept');
+      expect(response.body).toBeTruthy();
```

Comment:
It is an antipattern to assert truthy, when that is already implied by the below assertions. Not an issue though